### ISSUE TITLE
[Suggestion] Entangled Memory Cards (Items)

### ISSUE THREAD

**Issue by BrisingrAerowing** [maruohon/enderutilities#58]
How about adding an entangled memory card for items? The crafting recipe would output two cards that share the same inventory data.

The reason I suggest this is that I use the Handy Bag to automatically pick up mob and block drops, and having to go back to the dimension my main Refined Storage system is in when the bag fills up is a hassle. I would have a handy chest with one of the cards in it connected to a RS importer, and the other card in my handy bag, so that when something gets picked up, it is automatically inserted into my storage system through the handy chest.

Basically this is an Ender chest with automation and automatic pickup capabilities.

**Comment by maruohon** [comment]
I might consider this when/if I get to re-writing my item-based inventories to not be stored directly in the item's NBT data, but instead in an external inventory handler.

As to your use case, does the Item Pickup Manager linked to some form of an external inventory connected to your RS system not work like you would like?

**Comment by BrisingrAerowing** [comment]
I wasn't aware of the Item Pickup Manager. I haven't played much recently, I have mostly been watching Zedaph on Hermitcraft (and doing chores). I'll have to take a look at that.

**Comment by BrisingrAerowing** [comment]
The item pickup manager works perfectly!
